optimism/src/keccak/constraints.rs

```diff
@@ -21,8 +23,6 @@ use kimchi::{
 };
 use kimchi_msm::columns::ColumnIndexer;
 
-use super::interpreter::KeccakInterpreter;
-
 /// This struct contains all that needs to be kept track of during the execution of the Keccak step interpreter
 #[derive(Clone, Debug)]
 pub struct Env<Fp> {
```

Comment:
What do you think about calling this one `ConstraintEnv` and the other `WitnessEnv`?

Reply:
it is called like that from outside when accessing each of them (main), I just followed what appears in MIPS